src/game/objects/har.c

```diff
@@ -2641,22 +2641,40 @@ int har_create(object *obj, af *af_data, int dir, int har_id, int pilot_id, int
         power = obj->gs->match_settings.power1;
     }
 
-    // see https://www.omf2097.com/wiki/doku.php?id=omf2097:stats
-    float power_multiplier = 2.25 - 0.25 * power;
+    // cheap way to check if we're in tournament mode
+    bool is_tournament = pilot->photo != NULL;
+
+    float jump_multiplier = 1.0;
+    float vitality_multiplier = 1.0;
+    float power_multiplier = 1.0;
+    if(!is_tournament) {
+        jump_multiplier = obj->gs->match_settings.jump_height / 100.0;
```

Comment:
Match settings doesn't fill these in to the configured values in tournament mode already, it forces values for the settings not allowed in tournament mode, see mechlab_create.

Reply:
But I guess since there's some other tournament specific calculations here, it doesn't really matter.